src/js/components/ChatFoot/ChatFoot.test.js

```diff
@@ -1,12 +1,39 @@
 import { mount } from 'enzyme';
 import React from 'react';
+import { GlobalContext } from '../../contexts/GlobalContextWrapper';
+import Config from '../../utilities/Config';
 import ChatFoot from './ChatFoot';
 
 describe('ChatFoot', () => {
   const foo = 'foo';
+  let baz = '';
+  function bar(id, input) {
+    baz = input;
+  }
+
+  const fooConfiguration = {
+    [foo]: {
+      messageLog: [
+        {
+          name: foo,
+          timeStamp: foo,
+          body: foo,
```

Comment:
I'd avoid always using `foo` because if you accidentally use the `timeStamp` property instead of `body`, the test won't even notice. Try to have test values within the same test be differentiated

Reply:
Good point